src/contributing/release_process.md

```diff
@@ -127,6 +127,47 @@ perform whatever testing is necessary to verify that the new feature has reached
 quality level for the release. Systems integrators and known adopters of Parsec will also be
 notified that a new release is available for integration testing.
 
+As part of the release testing, we run the end-to-end tests on a RaspberryPi for the following
+backends:
+
+- mbed crypto
+- TPM
+- Nitrokey HSM
+
+Further platforms and backends will be added to this in future releases. The results of the release
+testing are critical in determining if the parsec service and the tool is ready for release, and
+whether it meets the requirements and expectations of the end users.
+
+The following failures are expected in the RaspberryPi platform for the specific backends.
+
+- mbed-crypto: All tests are passing
+- PKCS11: 17 failures
+   - asym_encrypt_and_decrypt_rsa_pkcs
+   - asym_encrypt_decrypt_rsa_pkcs_different_keys
+   - simple_asym_encrypt_rsa_oaep_pkcs11
+   - simple_asym_encrypt_rsa_pkcs
+   - asym_verify_fail_ecc_sha256
+   - fail_verify_hash2_ecc
+   - fail_verify_hash_ecc
+   - fail_verify_hash_rsa
+   - sign_verify_hash_ecc
+   - simple_verify_hash_ecc
+   - rsa_encrypt_use_check
+   - check_public_ecc_export_format
+   - import_and_export_ecc_public_key
+   - import_and_export_ecc_public_key_by_export_public_key_fn
+   - check_format_import_ecc
+   - import_ecc_key
+   - import_ecc_key_twice
+- TPM: 5 failures
+   - simple_asym_encrypt_rsa_pkcs
+   - activate_credential_bad_data
+   - activate_credential_ecc
+   - activate_credential_rsa
+   - check_name
+
+The new release candidate should ideally not introduce any regressions.
```

Comment:
Language feels a bit vague here: "ideally not introduce any regressions". What do we really mean, and what is the process? Let's try to tighten this up. Perhaps we need to say that any regressions will result in the creation of a new GH issue and will be flagged to the community for assessment as to whether it is a blocking issue for the release, during the RC phase?

Reply:
Yes, I agree with your point. That should be clear as to what we need to do in case of any regressions. Will update this line with your suggestion.